Gems/Atom/Feature/Common/Code/Source/PostProcessing/BlendColorGradingLutsPass.cpp

```diff
@@ -417,7 +417,7 @@ namespace AZ
             if (scene)
             {
                 PostProcessFeatureProcessor* fp = scene->GetFeatureProcessor<PostProcessFeatureProcessor>();
-                AZ::RPI::ViewPtr view = scene->GetDefaultRenderPipeline()->GetDefaultView();
+                AZ::RPI::ViewPtr view = scene->GetDefaultRenderPipeline()->GetFirstView(GetPipelineViewTag());
```

Comment:
Maybe irrelevant to your PR, it probably shouldn't use GetDefaultRenderPipeline() but use m_pipeline

Reply:
changed, here and for the other instances as well